src/Models/OptionValue.php

```diff
@@ -14,17 +14,13 @@ public static function getCachedByOptionId(int $optionId): string
     {
         $cacheKey = 'optionvalue.' . config('rapidez.store') . '.' . $optionId;
 
-        if (! $optionValue = config('cache.app.' . $cacheKey)) {
-            $optionValue = Cache::rememberForever($cacheKey, function () use ($optionId) {
-                return html_entity_decode(self::where('option_id', $optionId)
-                    ->whereIn('store_id', [config('rapidez.store'), 0])
-                    ->orderByDesc('store_id')
-                    ->first('value')
-                    ->value ?? false);
-            });
-
-            config(['cache.app.' . $cacheKey => $optionValue]);
-        }
+        $optionValue = once(fn () => Cache::rememberForever($cacheKey, function () use ($optionId) {
```

Comment:
Does this still makes sense as we're caching a lot of keys for every option id within Redis? Maybe it's faster to combine everything in one key and filter on that?

Reply:
You're right, i've moved it into a single cache